lib/Kconfig.kfence

```diff
@@ -5,6 +5,7 @@ menuconfig KFENCE
 	bool "KFENCE: low-overhead use-after-free and buffer-overflow detector"
 	depends on HAVE_ARCH_KFENCE && (!KASAN || EXPERT) && (SLAB || SLUB)
 	depends on JUMP_LABEL # To ensure performance, require jump labels
+	select STACKTRACE
```

Comment:
Didn't know one could unselect it.

Reply:
Yeah, make tinyconfig did not select it. (Some eventual random config would have hit this and we'd have gotten test-robot complaints.)